tensorflow/lite/arena_planner.cc

```diff
@@ -153,14 +153,14 @@ TfLiteStatus ArenaPlanner::PlanAllocations() {
     }
   }
   // Go through the graph in execution order.
-  for (int i = 0; i < graph_info_->num_nodes(); ++i) {
+  for (size_t i = 0; i < graph_info_->num_nodes(); ++i) {
```

Comment:
since j is an int, and we generally use int indices elswhere, should we just keep this as an int and use `static_cast<int>(graph_info_->num_nodes())?

Reply:
@jdduke , i have updated the code as per comments, kindly check and update.

Regards
Amit